spec/requests/services_spec.rb

```diff
@@ -267,7 +267,7 @@
                                                    {"action" => "remove", "path" => "description"},
                                                    {"action" => "add",    "path" => "display", "value" => true}])
```

Comment:
Does this one need to change too?

Reply:
it is not needed for fixing CI failure.

(but maybe it is needed for removing deprecation warning)